R/data.R

```diff
@@ -25,33 +28,28 @@
 #' incidence and/or observed cases is often undesirably low for testing. Many
 #' empirical datasets are much noisier. As a result, models built with these
 #' settings in mind can perform poorly on this dataset or fail to converge. We
-#' manually add observation noise with `rnbinom(299, mu =
-#' stochastic_sir_rt[["obs_cases"]], size = 10)` and the random seed 123456 and
-#' store it in the `obs_incidence` column.
+#' manually add observation noise and a day-of-week reporting effect.
 #'
 #' @name stochastic_sir_rt
-#' @format `stochastic_sir_rt` A data frame with 301 rows and 12 columns:
+#' @format `stochastic_sir_rt` A data frame with 299 rows and 6 columns:
 #' \describe{
-#'    \item{time}{Timestep of the discrete-time stochastic SEIR simulation}
-#'    \item{date}{Added from the original Gostic, 2020 dataset. A date
-#'    corresponding to the assigned `time`. Arbitrarily starts on January 1st,
-#'    2023.}
-#'    \item{S, E, I, R}{The realized state of the stochastic SEIR system}
-#'    \item{dS, dEI, DIR}{The stochastic transition between compartments}
-#'    \item{incidence}{The true incidence in the `I` compartment at time t}
-#'    \item{obs_cases}{The observed number of cases at time t from
-#'    forward-convolved incidence.}
-#'    \item{obs_incidence}{Added from the original Gostic, 2020 dataset. The
-#'     `incidence` column with added negative-binomial observation noise.
-#'     Created with `set.seed(123456)` and the call
-#'      `rnbinom(299, mu = [["incidence"]], size = 10)` Useful for
-#'       testing.}
-#'    \item{true_r0}{The initial R0 of the system (i.e., 2)}
-#'    \item{true_rt}{The known, true Rt of the epidemic system}
+#'    \item{reference_date}{The date the cases were observed.}
+#'    \item{true_rt}{The known, true Rt of the epidemic system.}
+#'    \item{dow}{The magnitude of the day-of-week effect added to the log of `incidence`.}
+#'    \item{incidence}{The true number of cases occurring on the `date` in the
```

Comment:
```suggestion
#'    \item{incidence}{The true number of infections occurring on the `date` in the
```

Is this correct?

Reply:
It looks like you dropped some columns here. Can you go back and make sure the documentation is clear about infections vs. cases (defined as infections convolved with a delay). I'd also consider whether you want DOW effects to modify infections or cases.

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/91/commits/90f5f700e686fcd473deca09ab3db43fa7c0b7fe